src/main/java/com/google/cloud/tools/maven/StageMojo.java

```diff
@@ -247,7 +244,11 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
-  protected boolean isStandardStaging() {
+  protected boolean isStandardStaging() throws MojoExecutionException {
+    if (appengineWebXml == null) {
+      appengineWebXml =
+          new AppEngineWebXml(new File(sourceDirectory.toString() + "/WEB-INF/appengine-web.xml"));
```

Comment:
Should probably use path resolution to build the path rather than string concatenation.